pkgs/tools/security/wad/default.nix

```diff
@@ -0,0 +1,46 @@
+{ fetchPypi
+, mock
+, pytestCheckHook
+, python3
+, six
+, stdenv
+}:
```

Comment:
Calling this package with `python3Packages.callPackage` allows to python access functions and packages directly:

```suggestion
{ lib
, buildPythonApplication
, fetchPypi
, mock
, pytestCheckHook
, six
}:
```

Reply:
Changed